packages/paste-website/src/components/ContactUsMenu.tsx

```diff
@@ -16,33 +18,45 @@ export const ContactUsMenu: React.FC<{}> = () => {
   };
 
   return (
-    <>
-      <MenuButton {...menu} variant="link">
-        Contact us <ChevronDownIcon decorative />
-      </MenuButton>
-      <Menu {...menu} aria-label="Contact us">
-        <MenuItem
-          {...menu}
-          href="https://github.com/twilio-labs/paste/discussions"
-          onClick={() => handleClick('Contact Menu', 'Ask a question')}
-        >
-          Ask a question
-        </MenuItem>
-        <MenuItem
-          {...menu}
-          href="https://github.com/twilio-labs/paste/discussions"
-          onClick={() => handleClick('Contact Menu', 'Request a feature')}
-        >
-          Request a feature
-        </MenuItem>
-        <MenuItem
-          {...menu}
-          href="https://github.com/twilio-labs/paste/issues/new?assignees=&labels=Type%3A+Bug&template=bug_report.md&title="
-          onClick={() => handleClick('Contact Menu', 'Report a bug')}
-        >
-          Report a bug
-        </MenuItem>
-      </Menu>
-    </>
+    <Box position="fixed" bottom={0} left={0}>
```

Comment:
When hovering over a nav item, this kind of bugs me

<img width="478" alt="Screen Shot 2020-10-05 at 16 30 36" src="https://user-images.githubusercontent.com/368249/95142306-36898300-0728-11eb-83b6-20ab6eb3167e.png">

Reply:
I've moved the button up a bit so its no longer covered by that text.